packages/btfs-http-client/examples/upload-file-via-browser/src/App.js

```diff
@@ -46,47 +89,206 @@ class App extends React.Component {
     }
   }
 
+  async getStatus(event) {
+    let input  = {
+      SessionId: this.state.added_session_id,
+    }
+    const statusResponse = this.btfs.statusSign(input, {})
+    try {
+      for await (const response of statusResponse) {
+        this.state.added_session_status = response.Status
+        this.state.added_status_response =  response.Message
+      }
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  async sign(event, data) {
+    let input  = {
+      SessionId: this.state.added_session_id,
+      SessionStatus: this.state.added_session_status,
+      Hash: this.state.added_file_hash,
+      Unsigned: data,
+      TimeNonce: this.time,
+      PrivKey: config.PrivKey,
+      PeerID: config.PeerID
+
+    }
+    const signResponse = this.btfs.sign(input, {})
+    try {
+      for await (const response of signResponse) {
+      }
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  async getBatch(event) {
+    let input  = {
+      SessionId: this.state.added_session_id,
+      SessionStatus: this.state.added_session_status,
+      Hash: this.state.added_file_hash,
+      TimeNonce: this.time,
+      PrivKey: config.PrivKey,
+      PeerID: config.PeerID
+    }
+    const batchResponse = this.btfs.getBatch(input, {})
+    try {
+      for await (const response of batchResponse) {
+        input.Contracts = (response).Contracts
+        const signBatchResponse = await this.btfs.signBatch(input, {})
+        for await (const response of signBatchResponse ){
+        }
+      }
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  async getUnsignedData(event) {
+    let input  = {
+      SessionId: this.state.added_session_id,
+      SessionStatus: this.state.added_session_status,
+      Hash: this.state.added_file_hash,
+      TimeNonce: this.time,
+      PrivKey: config.PrivKey,
+      PeerID: config.PeerID
+    }
+    const unsignedDataResponse = this.btfs.getUnsigned(input, {})
+    try {
+      for await (const response of unsignedDataResponse) {
+        input.Unsigned = response.Unsigned
+        input.Opcode = response.Opcode
+        input.Price = response.Price
+        const unsignedDataResponse = await this.btfs.sign(input, {}, {})
+        for await (const response of unsignedDataResponse ){
+        }
+      }
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  displayStatus(event) {
+    this.statusTimer = setInterval(() => {
+      this.getStatus(event)
+      if (errorCount > 9){
+        //after 10 error, cancel both timers
+        clearTimeout(this.statusTimer)
+        clearTimeout(this.stateTimer)
+      }
+    }, 1000)
+
+    this.stateTimer = setInterval(async () => {
+      console.log(this.state.added_session_status)
```

Comment:
we usually remove all console.log when checking in code, except when error

Reply:
okay, will do.